kukur/exceptions.py

```diff
@@ -17,6 +17,20 @@ def __init__(self, message: str):
         KukurException.__init__(self, f"invalid data: {message}")
 
 
+class InvalidPathException(KukurException):
+    """Raised when the path to the data is invalid."""
+
+    def __init__(self, message: str):
+        KukurException.__init__(self, f"invalid path: {message}")
```

Comment:
Just one exception for the case when a series is not found is enough. It should just not be called InvalidDataException.